src/Core/src/ImageSources/ImageSourcePaint.cs

```diff
@@ -0,0 +1,19 @@
+﻿#nullable enable
+using Microsoft.Maui.Graphics;
+
+namespace Microsoft.Maui
+{
+	class ImageSourcePaint : Paint
```

Comment:
This can't really be moved into Maui.Graphics as it wraps a Maui UI type. This is fine here - either public or internal - as it is not actually used by Maui.Graphics at all.

There is the `ImagePaint` still, but that represents an image already loaded into memory. The pixel bits. This new type represents the potential image and something must load it.